tests/backends/sqlite/tests.py

```diff
@@ -116,6 +116,26 @@ def test_check_database_version_supported(self, mocked_get_database_version):
             connection.check_database_version_supported()
         self.assertTrue(mocked_get_database_version.called)
 
+    def test_init_command(self):
+        settings_dict = {
+            "default": {
+                "ENGINE": "django.db.backends.sqlite3",
+                "NAME": ":memory:",
+                "OPTIONS": {
+                    "init_command": "PRAGMA synchronous=3;",
```

Comment:
I noticed that it won't work for multiple PRAGMA statements:
```
sqlite3.Warning: You can only execute one statement at a time.
```
I think we should change it to the list.